chatkit-core/src/main/kotlin/com/pusher/chatkit/presence/PresenceSubscriptionEvent.kt

```diff
@@ -1,10 +1,11 @@
 package com.pusher.chatkit.presence
 
+import com.pusher.chatkit.presence.api.UserPresenceApiType
 import elements.Error
 
-typealias PresenceSubscriptionConsumer = (PresenceSubscriptionEvent) -> Unit
+internal typealias PresenceSubscriptionConsumer = (PresenceSubscriptionEvent) -> Unit
 
-sealed class PresenceSubscriptionEvent {
-    internal data class PresenceUpdate(val presence: UserPresence) : PresenceSubscriptionEvent()
+internal sealed class PresenceSubscriptionEvent {
+    internal data class PresenceUpdate(val presence: UserPresenceApiType) : PresenceSubscriptionEvent()
     internal data class ErrorOccurred(val error: Error) : PresenceSubscriptionEvent()
```

Comment:
the error case can be removed and the whole event simplified to just: `internal data class PresenceSubscriptionEvent ` (no sealed hierarchy needed).

Reply:
👍 makes sense!